es-app/src/views/gamelist/ISimpleGameListView.cpp

```diff
@@ -93,6 +93,8 @@ bool ISimpleGameListView::input(InputConfig* config, Input input)
 				{
 					mCursorStack.push(cursor);
 					populateList(cursor->getChildrenListToDisplay());
+					FileData* cursor = getCursor();
+					setCursor(cursor);
```

Comment:
@zigurana - I think these two lines are what fixed the images/metadata not showing when opening the folder, if I recall correctly.